lib/measured/units/dimension.rb

```diff
@@ -0,0 +1,26 @@
+class Measured::Dimension < Measured::Measurable
```

Comment:
`dimension of a mathematical space (or object) is informally defined as the minimum number of coordinates needed to specify any point within it` https://en.wikipedia.org/wiki/Dimension